packages/near-membrane-base/src/environment.ts

```diff
@@ -76,12 +80,9 @@ export class VirtualEnvironment {
         if (options === undefined) {
             throw new ErrorCtor(`Missing VirtualEnvironmentOptions options bag.`);
         }
-        const {
-            blueConnector: localInit,
-            redConnector: foreignInit,
-            distortionCallback,
-            support,
-        } = options;
+        const { blueConnector, redConnector, distortionCallback, support } = options;
+        this.blueConnector = blueConnector;
+        this.redConnector = redConnector;
```

Comment:
Do these need to be exposed? ☝️

Reply:
yeah, for multi-window... explained in the description of the PR, mostly because the distortion for multi-window will not be creating a new env via createVirtualEnvironment, but instead, it will create a reflection membrane by using the low level connectors, and those can be extracted from an existing env instance.